src/fswAlgorithms/pointCloudProcessing/cameraTriangulation/cameraTriangulation.cpp

```diff
@@ -126,9 +130,15 @@ void CameraTriangulation::readMessages()
     double up = resolutionX/2;
     double vp = resolutionY/2;
     // build inverse K^-1 of camera calibration matrix K
-    this->cameraCalibrationMatrixInverse << 1./dX, -alpha/(dX*dY), (alpha*vp - dY*up)/(dX*dY),
+    Eigen::Matrix3d Kinv;
+    Kinv << 1./dX, -alpha/(dX*dY), (alpha*vp - dY*up)/(dX*dY),
                                             0., 1./dY, -vp/dY,
                                             0., 0., 1.;
+    this->cameraCalibrationMatrixInverse = Kinv;
+
+    Eigen::MatrixXd S(2,3);
+    S << Eigen::Matrix2d::Identity(), Eigen::MatrixXd::Zero(2, 1);
+    this->Rx = Kinv*S.transpose()*this->Ru*S*Kinv.transpose();
```

Comment:
If this is the same code as in other parts of the module, I'd consider making this a method that can be called in each location

Reply:
This is the only part of the module with this code. I will mark as resolved.